pracinejr/Streamish
# added repository files for video and tested api on postman

#### Changes Made

1. Added file `VideoRepository` and `IVideoRepository` to `Repositories`.
2. checked the API on Postman
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout pr-repositories-setup
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
